homeassistant/components/climate/ecobee.py

```diff
@@ -236,10 +249,30 @@ def device_state_attributes(self):
             "fan_min_on_time": self.fan_min_on_time
         }
 
+    def is_vacation_on(self):
+        """Return true if vacation mode is on."""
+        mode = ''
```

Comment:
This comment applies to any such pattern looping over events. Do you want me to change this in all those situations, even if not related to this request?